scripts/helper.ts

```diff
@@ -48,16 +48,19 @@ function typeMap(type: string, impHolder: Map<string, Set<string>>): string {
     case 'RightHolder[]':
     case 'CodeType':
     case 'CodeType[]':
+    case 'execution_code_data':
+    case 'execution_code_data[]':
       typeName = formatName(type);
```

Comment:
@danielAlvess 
I think `formatName` function should automatically remove `[]` if there is one.

Reply:
I know, that is the reason why array type generated is not correct, it missing array brackets.